configsuite/types.py

```diff
@@ -129,7 +129,7 @@ def __call__(self, *args, **kwargs):
 
 
 BasicType = collections.namedtuple("Type", ["name", "validate"])
-Collection = collections.namedtuple("Type", ["name", "validate"])
+Collection = collections.namedtuple("Type", ["name", "validate", "create_empty"])
```

Comment:
The collections now have to offer a function to create an empty container instance.